### PR TITLE
Fixed Automate tab issue for litdb and VRT Action run configuration

### PR DESCRIPTION
**Pull request by arvindgh**

### CODE REVIEW

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs [hunk 1]
```diff
@@ -165,15 +166,23 @@ private void StartVRT()
                 foreach (var e in ae.InnerExceptions)
                 {
                     mAct.Error += e.Message;
+                    mAct.Error += " Please check VRT configuration.";
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:169]: change it to 'Please check VRT configurations from Configurations -> External Integrations -> VRT configurations'

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComaprePage.xaml [hunk 2]
```diff
@@ -50,7 +50,7 @@
         </DockPanel>
 
         <StackPanel Grid.Row="3" Orientation="Horizontal" Margin="0,10,0,0" x:Name="xVRTDiffTollerancePercent">
-            <Label  MinWidth="150px" Style="{StaticResource $LabelStyle}" Grid.Column="0" VerticalAlignment="Center" Content="Diff Tollerance Percent:"/>
```
Comment (Maheshkale447) [inline Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComaprePage.xaml:53]: Please correct speelling of compare in this xaml filename

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComaprePage.xaml (hunk 2 above)
Comment (arvindgh) [reply Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComaprePage.xaml:53]: Done

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs (hunk 1 above)
Comment (arvindgh) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**